requirements.txt

```diff
@@ -1,4 +1,5 @@
-ethereum
 click
-coincurve
-web3>=4.0.0-beta.4
+rlp<1.0.0
+coincurve>=7.1.0,<8.0
+ethereum>=2.3.1,<3.0
+web3>=4.0.0b1
```

Comment:
You can use `web3>=4.0.0`.
Tests fail for `web3 < 4.0.0b13` (e.g. `4.0.0b11`) .

Reply:
okay I'll create an issue to update the version in libs and other projects as well